guests/automated_emails.py

```diff
@@ -48,8 +48,14 @@ def __init__(self, subject, template, filter, ident, **kwargs):
 AutomatedEmail(GuestGroup, 'It\'s time to send us your info for {EVENT_NAME}!', 'guest_checklist_announce.html',
                lambda g: g.group_type == c.GUEST, ident='guest_checklist_inquiry', sender=c.GUEST_EMAIL)
 
-GuestEmail('Reminder: Please complete your Guest Checklist for {EVENT_NAME}!', 'guest_checklist_reminder.html',
-           lambda g: not g.checklist_completed, when=days_before(14, c.GUEST_BIO_DEADLINE), ident='guest_reminder_1')
-
-GuestEmail('Have you forgotten anything? Your {EVENT_NAME} Guest Checklist needs you!', 'guest_checklist_reminder.html',
-           lambda g: not g.checklist_completed, when=days_before(7, c.GUEST_BIO_DEADLINE), ident='guest_reminder_2')
+GuestEmail('Reminder: Please complete your Guest Checklist for {EVENT_NAME}!',
+           'guest_checklist_reminder.html',
+           lambda g: not g.checklist_completed,
+           when=days_before(7, c.GUEST_BIO_DEADLINE - timedelta(days=7)),
```

Comment:
It looks like there's more changes here than there actually are, because I changed the line breaks.

The only thing that *ACTUALLY* changed is this:
```
when=days_before(14, c.GUEST_BIO_DEADLINE)
```
to this:
```
when=days_before(7, c.GUEST_BIO_DEADLINE - timedelta(days=7))
```